runtime/misc.c

```diff
@@ -51,7 +51,7 @@ void caml_failed_assert (char * expr, char_os * file_os, int line)
 {
   char* file = caml_stat_strdup_of_os(file_os);
   fprintf(stderr, "[%02d] file %s; line %d ### Assertion failed: %s\n",
-          (Caml_state_opt != NULL) ? Caml_state->id : -1, file, line, expr);
+          (Caml_state_opt != NULL) ? Caml_state_opt->id : -1, file, line, expr);
```

Comment:
The same conditional pattern occurs in `caml_gc_log` below (and nowhere else in the codebase). Could you also change it there for simplicity and consistency?

Reply:
OK, makes sense.  Done on the latest push.